### PR TITLE
Fix/hide archived beds for Orderportal

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds the possibility to hide archived beds from being shown to customers in the Orderportal

**How to prepare for test**:
- [x] ssh to hasta/clinical-db depending on type of change
- [x] install on stage:
`bash servers/resources/clinical-db.sclifelab.se/update-clinical-api-stage.sh [THIS-BRANCH-NAME]`

**How to test**:
- [x] login to the clinical-api-stage
- [x] archive a bed in the bed view
- [x] login to the Orderportal
- [x] create Balsamic order
- [x] add sample and open bed drop down

**Expected test outcome**:
- [x] check that the bed that was archived is not listed
- [x] Take a screenshot and attach or copy/paste the output

**Review:**
- [ ] code approved by
- [x] tests executed by @patrikgrenfeldt 
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
Test result:

Clinical-api:
image

OP:
image

👍

File: cg/store/api/findbasicdata.py [hunk 1]
```diff
@@ -24,25 +24,31 @@ def applications(self, *, category=None, archived=None):
...
+            application=application, version=version
+        )
         return query.first()
 
     def bed(self, name):
         """Find a bed by name."""
         return self.Bed.query.filter_by(name=name).first()
 
-    def beds(self):
+    def beds(self, hide_archived: bool = False):
```
Comment (mayabrandi) [inline cg/store/api/findbasicdata.py]: Easy to read. 👍

**Comment by patrikgrenfeldt** [comment]
Still works after refactoring: 
image
 👍

**Comment by patrikgrenfeldt** [comment]
Thank you for your quick review @mayabrandi !

File: cg/store/api/findbasicdata.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/store/api/findbasicdata.py]: Thanks!

**Comment by patrikgrenfeldt** [comment]
deployed on clinical-db

**Pull request merged**